tests/e2e/footer.spec.ts

```diff
@@ -0,0 +1,126 @@
+import { expect, test } from '@playwright/test'
+
+const footerLinks = [
+  {
+    linkName: 'Ambassador Program',
+    linkAddress:
+      'https://hello.kodadot.xyz/be-part-of-kodadot/kodadots-programs/ambassador-program',
+  },
+  {
+    linkName: 'Artist Ambassador',
+    linkAddress:
+      'https://hello.kodadot.xyz/be-part-of-kodadot/kodadots-programs/artist-ambassador-program',
+  },
+  {
+    linkName: 'Referral Program',
+    linkAddress:
+      'https://hello.kodadot.xyz/be-part-of-kodadot/kodadots-programs/artist-referral-program',
+  },
+  {
+    linkName: 'Developers',
+    linkAddress: 'https://developers.kodadot.xyz/',
+  },
+  {
+    linkName: 'FAQ',
+    linkAddress: 'https://hello.kodadot.xyz/ecosystem/faq',
+  },
+  {
+    linkName: 'Tutorial',
+    linkAddress: 'https://hello.kodadot.xyz/tutorial/wallet',
+  },
+  {
+    linkName: 'About',
+    linkAddress: 'https://hello.kodadot.xyz/about-us/who-are-we',
+  },
+  {
+    linkName: 'Careers',
+    linkAddress:
+      'https://hello.kodadot.xyz/be-part-of-kodadot/join-as-a-developer/hiring',
+  },
+  {
+    linkName: 'Press Kit',
+    linkAddress: 'https://github.com/kodadot/kodadot-presskit/tree/main/pre-v4',
+  },
+  {
+    linkName: 'MerchShop',
+    linkAddress: 'https://shop.kodadot.xyz/',
+  },
+]
+
+const footerSocialMediaLinks = [
+  {
+    linkName: 'Twitter',
+    linkAddress: 'https://twitter.com/KodaDot',
+  },
+  {
+    linkName: 'Discord',
+    linkAddress: 'https://discord.com/invite/u6ymnbz4PR',
+  },
+  {
+    linkName: 'Substack',
+    linkAddress: 'https://kodadot.substack.com',
+  },
+  //{
+  //  linkName: 'Medium',
+  //  linkAddress: 'https://medium.com/kodadot',
+  //},
+  {
+    linkName: 'Youtube',
+    linkAddress: 'https://www.youtube.com/channel/UCEULduld5NrqOL49k1KVjoA',
+  },
+  //{
+  //  linkName: 'Instagram',
+  //  linkAddress: 'https://instagram.com/kodadot.xyz',
+  // },
+  {
+    linkName: 'Reddit',
+    linkAddress: 'https://www.reddit.com/r/KodaDot/',
+  },
+]
+
+test('Check Footer Subscription', async ({ page }) => {
+  await page.goto('/')
+  const footerSubscribe = page.getByTestId('footer-subscribe')
+  await page.getByPlaceholder('jane.doe@kodadot.xyz').fill('a')
+  await footerSubscribe.locator('button').click()
+  await expect(footerSubscribe.locator('.error')).toBeVisible()
+})
+
+test('Check Footer links', async ({ page }) => {
+  let i = 0
+  await page.goto('/')
+  for (const data of footerLinks) {
```

Comment:
code sonar complain about 3 code smells related to these loops

`data` is not used

Reply:
any tips on how can i solve that?

Reply:
```ts
for (const [index, _] of footerLinks.entries())
```

honestly, why have we disallowed regular for loops in eslint?